src/components/FeatureSetsPanel/FeatureSetsPanelSchema/featureSetsPanelSchema.scss

```diff
@@ -15,6 +15,6 @@
   }
 
   &__description {
-    margin-bottom: 10px;
+    margin-bottom: 1rem;
```

Comment:
If I'm not mistaken you deleted the class `schema__description` . So this style is redundant